docs/clusters/quick-install.md

```diff
@@ -1,39 +1,70 @@
 ---
 id: quick-install
-title: How to quickly install a Temporal Cluster for testing and local development
-sidebar_label: Quick install
-description: There are four ways to quickly install and run a Temporal Cluster.
+title: How to quickly install a Temporal Cluster
+sidebar_label: Install Temporal Cluster
+description: Quickly install and run a Temporal Cluster in various ways.
 ---
 
-Whenever you are developing Temporal Applications, you'll want to have a [Temporal Cluster](/concepts/what-is-a-temporal-cluster) up and running.
-You can interact with a Cluster through [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl) commands.
+The following section list different methods of deploying your [Temporal Clusters](/concepts/what-is-a-temporal-cluster) locally, so that you can use and interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl) commands to test and develop applications.
 
-There are four ways to quickly install and run a Temporal Cluster:
-
-- [Docker](#docker-compose): Using Docker Compose makes it easy to develop your Temporal Application locally.
+- [Temporalite](#temporalite): This distribution of Temporal runs as a single process with zero runtime dependencies.
+- [Docker](#docker-compose): Using Docker Compose makes it easy to develop your Temporal Application.
```

Comment:
```suggestion
- [Docker](#docker-compose): Using Docker Compose simplifies developing your Temporal Application.
```
A claim that something is "easy" is subject to wide interpretation. See https://developers.google.com/style/word-list?hl=en#easy.